xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentAccess_support.xsd

```diff
@@ -69,7 +69,7 @@ Rail transport, Roads and Road transport
 		</xsd:annotation>
 		<xsd:restriction base="InstalledEquipmentIdType"/>
 	</xsd:simpleType>
-	<xsd:element name="AccessEquipmentRef" type="AccessEquipmentRefStructure" substitutionGroup="EquipmentRef">
```

Comment:
This is new. It currently works with EquipmentRef (as this is the more generalized class). However, to be very in line, whe should probably keep this change. How pedantic do we want to be.